docs/getting-started.md

```diff
@@ -488,3 +491,121 @@ Now you can create a key, using this security policy, and access your API.
 > We are continously adding support for new features which you can track [here](./policies.md)
 
 
+
+
+
+
+
+## Publish an API to the Tyk Portal
+
+Asumming that you have created the httpbin example API using the steps above, we can easily publish it to the Tyk Portal by applying a few specs:
+1. SecurityPolicy spec
+2. APIDescription spec
+3. PortalAPICatalogue spec
+
+
+### 1. Adding a security policy via Tyk Operator
+
+When you publish an API to the Portal, Tyk actually publishes a way for developers to enrol in a policy, not into the API directly. Therefore, we must also apply a SecurityPolicy CRD before proceeding with the publishing.
+
+For this we can create a file called `securityPolicyHttpBin.yaml` with the following contents :
+
+```yaml
+apiVersion: tyk.tyk.io/v1alpha1
+kind: SecurityPolicy
+metadata:
+ name: standard-pol
+spec:
+ name: standard-pol
+ active: true
+ state: active
+ access_rights_array:
+ - name: httpbin
+   namespace: default
+   versions:
+     - Default
+```
+
+and then run:
+
+```bash
+kubectl apply -f securityPolicyHttpBin.yaml
+```
```

Comment:
This would look much neater using heredoc.

```bash
kubectl apply -f - <<EOF
apiVersion: tyk.tyk.io/v1alpha1
kind: SecurityPolicy
metadata:
 name: standard-pol
spec:
 name: standard-pol
 active: true
 state: active
 access_rights_array:
 - name: httpbin
   namespace: default
   versions:
     - Default
EOF
```